File: Benchmarks/enterprise/python/django/testcode/benchmark_test_75983.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest75983(request):
    cookie_value = request.COOKIES.get('session_token', '')
    data = (lambda v: v.strip())(cookie_value)
    resp = JsonResponse({'status': 'ok'})
    resp.set_cookie('session', str(data), max_age=86400)
    return resp

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest75983.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest75983 {

    private static String normalize(String v) { return v.strip(); }

    @GetMapping("/BenchmarkTest75983/{pathId}")
    public void BenchmarkTest75983(@PathVariable("pathId") String pathId, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String pathValue = pathId;
        String data = normalize(pathValue);
        new ProcessBuilder("python3", "-c", data).start().waitFor(5, java.util.concurrent.TimeUnit.SECONDS);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
